### PR TITLE
fix(swap): don't add/sub fee for swap orders

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Context: https://cowservices.slack.com/archives/C036G0J90BU/p1703156515196189

This is a follow up of fee=zero feature.
The amount from Quote API already took into account the fee amount, so we don't need to substract it again.
The only this we should do is adding/substracting slippage.

image

  # To Test

In console you can find logs with prefix '[SWAP PARAMS]'.
`[SWAP PARAMS] quote` - displays what we got from quote
`[SWAP PARAMS] amounts to sign` - displays what we are going to sign

The simpliest way how to test:
1. Set slippage to 0
2. What you sign in your wallet must be the same what you got from Quote API
3. Set slippage to 50%
4. What you sign in your wallet must be 50% from what you got from Quote API

### CODE REVIEW

File: apps/cowswap-frontend/src/common/hooks/featureFlags/useFeatureFlags.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import { useFlags } from 'launchdarkly-react-client-sdk'
-
+// TODO: revert this change, this is just for testing
 export function useFeatureFlags() {
-  return useFlags()
+  return { swapZeroFee: true } as any
```
Comment (shoom3301) [inline apps/cowswap-frontend/src/common/hooks/featureFlags/useFeatureFlags.ts:3]: TODO: revert this change, this is just for testing

**Review by alfetopito — APPROVED** [review]
